openstack/datadog_checks/openstack/openstack.py

```diff
@@ -1176,6 +1176,7 @@ def ensure_auth_scope(self, instance):
 
         return instance_scope
 
+    @add_tracing
```

Comment:
nit: can we rename it to something like `trace` or `traced` (depending on which style you prefer)

Reply:
@ofek Had a similar review here - https://github.com/DataDog/integrations-core/pull/2079#discussion_r217894019 I was hesitant to name it the same thing as an external function from the tracing library, I'll go with `traced` then.